std/range/package.d

```diff
@@ -3183,13 +3183,17 @@ pure @safe nothrow unittest
 }
 
 /**
-Create a range which repeats one value forever.
+Create a range which repeats one value.
 
 Params:
-    value = the value to repeat
+    value = the _value to repeat
+    n = the number of times to repeat `value`
```

Comment:
You are a bit inconsistent here (_value vs. `value`)

Reply:
It's the difference between referring to the parameter value (\`value\`) vs the concept of a value (_value).